chromium_src/chrome/browser/ui/views/permissions/permission_prompt_factory.cc

```diff
@@ -4,10 +4,12 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "base/logging.h"
+#include "brave/components/brave_wallet/common/buildflags/buildflags.h"
 #include "chrome/browser/ui/permission_bubble/permission_prompt.h"
 #include "components/permissions/request_type.h"
 
-#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
+#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS) && \
+    BUILDFLAG(ENABLE_BRAVE_WALLET)
 #include "brave/browser/ui/views/permission_bubble/brave_wallet_permission_prompt_impl.h"
 #endif
```

Comment:
`#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)` should be always true in this file, might be removed at all